lib/event_class.rb

```diff
@@ -0,0 +1,30 @@
+require 'pry'
+
+class Event
```

Comment:
Keep an eye on how you are naming your files - the convention is class.rb so this file should be named `event.rb`